authentication/provider/service/user_profile_service.go

```diff
@@ -0,0 +1,73 @@
+package service
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/fabric8-services/fabric8-auth/application/service"
+	"github.com/fabric8-services/fabric8-auth/application/service/base"
+	servicecontext "github.com/fabric8-services/fabric8-auth/application/service/context"
+	"github.com/fabric8-services/fabric8-auth/authentication/provider"
+
+	"github.com/fabric8-services/fabric8-auth/errors"
+	"github.com/fabric8-services/fabric8-auth/log"
+	"github.com/fabric8-services/fabric8-auth/rest"
+
+	errs "github.com/pkg/errors"
+)
+
+// userProfileService is the default implementation for UserProfileService
+type userProfileService struct {
+	base.BaseService
+	client *http.Client
+}
+
+// NewUserProfileService creates a new UserProfileService
+func NewUserProfileService(context servicecontext.ServiceContext) service.UserProfileService {
+	return &userProfileService{
+		client: http.DefaultClient,
+	}
+}
+
+//Get gets the user profile information from Oauth provider
+func (s *userProfileService) Get(ctx context.Context, accessToken string, profileURL string) (*provider.OAuthUserProfileResponse, error) {
```

Comment:
It's weird but codecove doesn't show any coverage for that service. Can you make sure the test is run? And what's wrong with the reports again..